tests/para-tests-no-ci/test-xcm-para.ts

```diff
@@ -384,7 +500,7 @@ describeParachain(
 
       // Get Pallet balances index
       const metadata = await parachainOne.rpc.state.getMetadata();
-      const palletIndex = (metadata.asLatest.toHuman().modules as Array<any>).find((pallet) => {
+      const palletIndex = (metadata.asLatest.toHuman().pallets as Array<any>).find((pallet) => {
```

Comment:
a leftover `toHuman`

Reply:
here is fine, as long as is not in the balances I think its ok